api/resource/factory.go

```diff
@@ -196,7 +196,8 @@ func (rf *Factory) inlineAnyEmbeddedLists(
 		}
 		items, ok := m["items"]
 		if !ok {
-			// treat as an empty list
+			//list is empty nothing to replace node with
+			result = append(result, n0)
```

Comment:
I'm not sure that this is the correct fix, and I am worried this could introduce unintended behaviour for actual embedded lists. Open to suggestions for knowing when the node is an embedded list.